net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -266,7 +266,7 @@ function quagga_ospfd_install_conf() {
 
 	/* Make zebra config */
 	if (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['zebra'])
-	    && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['zebra'])) {
+		&& !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['zebra'])) {
```

Comment:
These should be [indented with four spaces](https://www.netgate.com/docs/pfsense/development/developer-style-guide.html#indent-style)